src/Artemis.UI/Screens/Debugger/Tabs/Logs/LogsDebugView.axaml.cs

```diff
@@ -14,4 +18,10 @@ private void InitializeComponent()
     {
         AvaloniaXamlLoader.Load(this);
     }
+
+    private void OnTextChanged(object? sender, EventArgs e)
+    {
+        if (sender is TextEditor te)
+            te.ScrollToEnd();
```

Comment:
Maybe it's a good idea to only scroll to the bottom if the user was already at the bottom (or very near to) before the message got added.

That way you can scroll up to read a message without being thrown down all the time

Reply:
This is doable, i'm currently trying to figure out how far a user has to scroll up before this matters. Currently if you get a large exception with a 10 line stacktrace, it will not scroll dowm, because the cutoff i set is roughly ~2 lines high.

Maybe i could try and check how many lines were added but i don't know if this is possible.

Reply:
Are you currently just looking at pixels? Then I'd say just take a little error margin and put it at 20px.

Reply:
I'm looking at `TextEditor.ExtentHeight` and `TextEditor.VerticalOffset`. I'm not sure what units they're in, but each line is roughly 800 units high. I'm now trying to figure out what it means.